docker/README.md

```diff
@@ -32,6 +32,8 @@ docker run --rm -it \
 
 To run with `rocker`:
 
+If you use `rocker<=0.2.9`, add an option of `--env NVIDIA_DRIVER_CAPABILITIES=""` or `--env NVIDIA_DRIVER_CAPABILITIES=compute,utility,graphics` to avoid the CUDA environment error. See [this issue](https://github.com/autowarefoundation/autoware/issues/2452) in more details.
```

Comment:
- Maybe you should use `for` instead of `in`.
- It's better to write `for more details` first.
  - https://developers.google.com/style/clause-order?hl=en

```suggestion
If you use `rocker<=0.2.9`, add an option of `--env NVIDIA_DRIVER_CAPABILITIES=""` or `--env NVIDIA_DRIVER_CAPABILITIES=compute,utility,graphics` to avoid the CUDA environment error. For more details, see [this issue](https://github.com/autowarefoundation/autoware/issues/2452).
```

Reply:
changed at [e4bbdc6](https://github.com/autowarefoundation/autoware/pull/2732/commits/e4bbdc6b04658ccc1cee6797a90dff9fa7f1319a)